src/miniscript/decode.rs

```diff
@@ -472,3 +482,10 @@ pub fn parse(tokens: &mut TokenIter) -> Result<Miniscript<bitcoin::PublicKey>, E
     assert_eq!(term.0.len(), 1);
     Ok(term.pop().unwrap())
 }
+
+fn isAndV(tokens: &mut TokenIter) -> bool {
```

Comment:
rustc warning: "convert the identifier to snake case: `is_and_v`"

Reply:
Oh nice, thanks